recipes/diffpy-structure/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
 
 test:
   source_files:
-    - conda-recipe/run_test.py
+    - diffpy.structure/conda-recipe/run_test.py
```

Comment:
`run_test.py` file should run by default when present in the recipe directory.  Just add it as a new file to this PR.
There should be no need to tweak meta.yaml.